Resources/Prototypes/_SpaceCats/Entities/Clothing/Belt/belts.yml

```
@@ -81,3 +81,24 @@
+  - type: Clothing
+    sprite: _SpaceCats/BridgeOfficer/BOWebbing.rsi
+  - type: Item
+    size: Large
+  - type: EntityTableContainerFill
+    containers:
+      storagebase: !type:NestedSelector
+        tableId: BeltSecurityEntityTable
+  - type: Storage
+    maxItemSize: Huge
+    grid:
+    - 0,0,5,3
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)